gtsfm/data_association/point3d_initializer.py

```diff
@@ -200,6 +199,35 @@ def triangulate(self, track_2d: SfmTrack2d) -> Tuple[Optional[SfmTrack], Optiona
 
         return track_3d, avg_track_reproj_error, is_cheirality_failure
 
+    def is_small_triangulation_angle(
+        self, point_3d: np.ndarray, camera_indices: List[int], angle_threshold_deg: float = 1.5
+    ) -> bool:
+        """Check if all pairs of cameras have a small triangulation angle.
```

Comment:
thanks for the PR, Ayush. Maybe you could add a URL ref here to where other libraries also do this?

Reply:
added them now